src/ai-bundle/src/Command/ChatCommand.php

```diff
@@ -0,0 +1,189 @@
+            if ($this->agentName && !isset($agentMap[$this->agentName])) {
+                $io->warning(\sprintf('Agent "%s" not found.', $this->agentName));
+            }
+
+            // Interactive selection
+            $question = new ChoiceQuestion(
+                'Please select an agent to chat with:',
+                array_keys($availableAgents),
+                0
+            );
+            $question->setErrorMessage('Agent %s is invalid.');
+
+            /** @var \Symfony\Component\Console\Helper\QuestionHelper $helper */
+            $helper = $this->getHelper('question');
+            $selectedAgent = $helper->ask($input, $output, $question);
+            if (!\is_string($selectedAgent)) {
+                throw new InvalidArgumentException('Invalid agent selection.');
+            }
+            $this->agentName = $selectedAgent;
+        }
+
+        // Set the selected agent
+        if (!isset($agentMap[$this->agentName])) {
+            throw new InvalidArgumentException(\sprintf('Agent "%s" not found.', $this->agentName));
+        }
+
+        $this->agent = $agentMap[$this->agentName];
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $io = new SymfonyStyle($input, $output);
+
+        $io->title(\sprintf('Chat with %s Agent', $this->agentName));
+        $io->info('Type your message and press Enter. Type "exit" or "quit" to end the conversation.');
+        $io->newLine();
+
+        $messages = new MessageBag();
+        $systemPromptDisplayed = false;
+
+        while (true) {
+            $userInput = $io->ask('You');
+
+            if (!\is_string($userInput)) {
+                continue;
+            }
+
+            if (\in_array(strtolower($userInput), ['exit', 'quit'], true)) {
+                $io->success('Goodbye!');
+                break;
+            }
+
+            $messages->add(Message::ofUser($userInput));
+
+            try {
+                $result = $this->agent->call($messages);
+
+                // Display system prompt after first successful call
+                if (!$systemPromptDisplayed && null !== ($systemMessage = $messages->getSystemMessage())) {
+                    $io->section('System Prompt');
+                    $io->block($systemMessage->content, null, 'fg=gray', ' ', true);
+                    $systemPromptDisplayed = true;
+                }
+
+                if ($result instanceof TextResult) {
+                    $io->write('<fg=yellow>Assistant</>:');
+                    $io->writeln('');
+                    $io->writeln($result->getContent());
+                    $io->newLine();
+
+                    $messages->add(Message::ofAssistant($result->getContent()));
+                } else {
+                    $io->error('Unexpected response type from agent');
+                }
+            } catch (\Exception $e) {
+                $io->error(\sprintf('Error: %s', $e->getMessage()));
```

Comment:
In verbose mode, it is useful to have the exception backtrace.

Reply:
done